docs/client-development/mandate.rst

```diff
@@ -0,0 +1,526 @@
+  ``Rol.indicatieMachtiging`` attribute. If it's blank, that means that the party initiates
+  their own case.
+* for eHerkenning cases, the employee details should be added to ``Rol.contactpersoonRol``
+  attributes.
+
+Validation rules
+^^^^^^^^^^^^^^^^
+
+* if ``representee`` is provided in ``authenticatieContext``, then:
+    * ``indicatieMachtiging`` **MUST** be set to "gemachtigde"
+    * ``mandate`` **MUST** be provided in the ``authenticatieContext``
+
+* if ``betrokkeneType`` is ``natuurlijk_persoon``, then ``Rol.authenticatieContext.source``
+  **MUST** be set to "digid"
+
+* if ``betrokkeneType`` is ``niet_natuurlijk_persoon`` or ``vestiging``, then
+  ``Rol.authenticatieContext.source`` **MUST** be set to "eherkenning"
+
+
+Example API calls below are provided with required fields and dummy data, the focus is on the shape of
+``authenticatieContext``.
+
+1. DigiD / Without mandate
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**DigiD - initiator**
+
+.. code:: http
+
+    POST /zaken/api/v1/rollen HTTP/1.1
+    Content-Type: application/json
+
+    {
+        "zaak": "http://example.com",
+        "betrokkeneType": "natuurlijk_persoon",
+        "roltype": "http://example.com/roltype-initiator",
+        "roltoelichting": "Created zaak",
+        "betrokkeneIdentificatie": {
+            "inpBsn": "123456782"
+        },
+        "authenticatieContext": {
+            "source": "digid",
+            "levelOfAssurance": "urn:oasis:names:tc:SAML:2.0:ac:classes:MobileTwoFactorContract"
+        }
+    }
+
+
+2. DigiD / With mandate
+^^^^^^^^^^^^^^^^^^^^^^^
+
+**DigiD - initiator**
+
+The authorizee is the "initiator" of the case (cleared up from VNG github), the representee
+is a "belanghebbende".
+
+.. code:: http
+
+    POST /zaken/api/v1/rollen HTTP/1.1
+    Content-Type: application/json
+
+    {
+        "zaak": "http://example.com",
+        "betrokkeneType": "natuurlijk_persoon",
+        "roltype": "http://example.com/roltype-initiator",
+        "roltoelichting": "Created zaak",
+        "indicatieMachtiging": "gemachtigde",
+        "betrokkeneIdentificatie": {
+            "inpBsn": "123456782"
+        },
+        "authenticatieContext": {
+            "source": "digid",
+            "levelOfAssurance": "urn:oasis:names:tc:SAML:2.0:ac:classes:MobileTwoFactorContract"
+            "representee": {
+                "identifierType": "bsn",
+                "identifier": "111222333"
+            },
+            "mandate": {
+                "services": [
+                    {"id": "5628edbd-333e-460d-8a69-8f083b8cf1b8"}
+                ]
+            }
+        }
+    }
+
+
+**DigiD - belanghebbende**
+
+.. code:: http
+
+    POST /zaken/api/v1/rollen HTTP/1.1
+    Content-Type: application/json
+
+    {
+        "zaak": "http://example.com",
+        "betrokkeneType": "natuurlijk_persoon",
+        "roltype": "http://example.com/roltype-belanghebbende",
+        "roltoelichting": "Voogd",
```

Comment:
```suggestion
        "roltoelichting": "Beneficiary",
```
a "Voogd" in Dutch is typically a person that acts in the absence of parents, so that sounds more like an authorizee rather then representee